datasets/xglue/README.md

```diff
@@ -0,0 +1,328 @@
+---
```

Comment:
It took me 3h to generate this. The yaml content is complete. The dataset card could be improved.